lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -366,32 +364,22 @@ def start_watch
       watcher.each do |notice|
         case notice.type
           when 'MODIFIED'
-            if notice.object.status.containerStatuses
```

Comment:
Thinking about it again, I think it's probably reasonable to keep as this does it now, relying on the cache key rather than a field that will almost always be set AFAIK. We can always look at it again if we see problems with it - I haven't, but I'm obviously not running large scale performance tests.